docs/tutorials/prowler-app-s3-integration.md

```diff
@@ -0,0 +1,334 @@
+# S3 Integration (Amazon S3)
+
+**Prowler App** allows users to automatically export scan results to Amazon S3 buckets, enabling seamless integration with existing data workflows and storage infrastructure. This comprehensive guide demonstrates how to configure and manage Amazon S3 integrations to streamline security finding management and reporting.
+
+## What Is the S3 Integration Feature?
+
+The S3 Integration feature enables users to:
+
+- **Automate scan result exports** to designated S3 buckets after each scan
+
+- **Configure multiple bucket destinations** for different providers or use cases
+
+- **Customize export paths** within buckets for organized storage
+
+- **Support multiple authentication methods** including IAM roles and static credentials
+
+- **Test connection reliability** to ensure proper configuration
+
+- **Manage integrations independently** with separate configuration and credential controls
+
+## Prerequisites
+
+Before configuring S3 Integration, ensure:
+
+- A user with a Role that has `MANAGE_INTEGRATIONS`
+
+- At least one cloud provider configured
+
+- Access to an Amazon S3 bucket with proper write permissions
+
+- AWS credentials (IAM role or access keys) with S3 write permissions
+
+- Understanding of AWS IAM policies and S3 bucket permissions
+
+## Required Permissions Setup
+
+Before configuring S3 Integration, the existing Prowler SaaS role (ProwlerProSaaSScanRole) requires additional permissions to write scan results to the designated S3 bucket.
```

Comment:
We can't talk about Prowler SaaS here as this is meant for Prowler, either self-hosted or cloud.

Regarding the role, as in Prowler Cloud is not mandatory, we need to explain what is, how to create it and how to update it if it's there already. This should be a section as role is an optional part of the credentials flow.

Reply:
Done.